pkg/onchain/website/handler.go

```diff
@@ -29,6 +29,26 @@ func handleMassaDomainRequest(writer http.ResponseWriter, reader *http.Request,
 	Request(writer, reader, rpcClient, addr, target)
 }
 
+// Redirects a given request to https.
+func redirectTLDToHTTPS(req *interceptor.Interceptor) {
```

Comment:
Why not using the same approach as here: https://github.com/massalabs/thyra-plugin-hello-world/blob/main/web/endpoints.go#L33 ?

Something like:

```go
openapi.NewCustomResponder(nil, map[string]string{"Location": "https://" + r.Host + r.RequestURI}, http.StatusPermanentRedirect)
```